_sass/3-layout/_global.scss

```diff
@@ -0,0 +1,103 @@
+@import "./0-base/reset";
+@import "./1-modules/mixins";
+@import "./1-modules/colors";
+@import "./1-modules/selectBox";
+@import "./1-modules/stickyAside";
+
+@import "./list";
+@import "./email";
+
+body {
+  text-rendering: optimizeLegibility;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  -moz-font-feature-settings: "kern", "liga", "clig", "calt";
+  -ms-font-feature-settings: "kern", "liga", "clig", "calt";
+  -webkit-font-feature-settings: "kern", "liga", "clig", "calt";
+  font-feature-settings: "kern", "liga", "clig", "calt";
+  background-color: var(--bg);
+  font-family: 'Source Code Pro', 'Helvetica', 'Arial', sans-serif;
+  font-size: 16px;
+  line-height: 1.6;
+  font-weight: 400;
+  color: var(--text);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+a {
+  color: inherit;
+  text-decoration: none;
+  border-bottom: 1px solid var(--link_subtle);
+
+  &:hover {
+    border-color: var(--subtle_);
+  }
+
+  &:focus {
+    outline: none;
+    border-bottom: 1px solid var(--text_subtle);
+  }
+}
+
+em,
+i {
+  font-weight: 400;
+  font-style: italic;
+}
+
+b,
+strong {
+  font-weight: 700;
+}
+
+h1 {
+  font-weight: 700;
+  font-size: 36px;
+  line-height: 1.2;
+
+  @include breakpoint(s) {
+    font-size: 30px;
+  }
+}
+
+h2 {
+  font-size: 24px;
+  line-height: 1.2;
+  font-weight: 400;
+  margin-top: $x_2;
+
+  @include breakpoint(s) {
+    font-size: 22px;
+  }
+}
+
+header,
+footer {
+  @include siteWidth;
+}
+
+p.divider {
+  &:before {
+    content: "";
+    display: block;
+    width: $x_6;
+    border-bottom: 1px solid var(--link_subtle);
+    margin: $x_2 0 $x_3;
+  }
+
+  line-height: 1.6;
+  max-width: 720px;
+  margin-bottom: $x_2;
+}
+
+// Sections and Layout =========================================================
```

Comment:
nit) this comment can probably just be removed IMO

Reply:
DONE!